tests/unit/pages/Editor.spec.js

```diff
@@ -188,9 +188,9 @@ describe("Editor.vue", () => {
 
     // use `wrapper.vm.__` to access the updated data variables inside the component
     expect(wrapper.vm.loadedPlioDetails.items).toStrictEqual(
-      dummyItemsCopy.data
```

Comment:
we should also remove the definition of `dummyItemsCopy` right?

Reply:
I'm also not sure why we needed to have a clone of `dummyItems`. Maybe @deepansh96 have some insights here?